base/libgit2/merge.jl

```diff
@@ -111,6 +111,29 @@ function ffmerge!(repo::GitRepo, ann::GitAnnotated)
 end
 
 # Merge changes into current head
```

Comment:
This comment seem a bit superfluous now that we have a complete docstring?

Reply:
I think they aren't harming anyone by hanging around. In general I'd rather err on the side of having more context rather than less...